raiutils/tests/test_model_utils.py

```diff
@@ -44,3 +53,9 @@ def test_forecaster(self):
         assert not is_classifier(forecaster)
         assert is_forecaster(forecaster)
         assert is_quantile_forecaster(forecaster)
+
+    def test_object_detection_model(self):
+        forecaster = ObjectDetectionModel()
```

Comment:
Maybe change the variable name here to object_detector here instead of forecaster?